packages/sdk/contractkit/src/wrappers/Election.test.ts

```diff
@@ -0,0 +1,257 @@
+  ) => {
+    await setupGroup(groupAccount)
+    await setupValidator(validatorAccount)
+    await validators.affiliate(groupAccount).sendAndWaitForReceipt({ from: validatorAccount })
+    await (await validators.addMember(groupAccount, validatorAccount)).sendAndWaitForReceipt({
+      from: groupAccount,
+    })
+  }
+
+  const activateAndVote = async (groupAccount: string, userAccount: string, amount: BigNumber) => {
+    await (await election.vote(groupAccount, amount)).sendAndWaitForReceipt({ from: userAccount })
+    await mineToNextEpoch(web3)
+
+    const txList = await election.activate(userAccount)
+
+    let promises: Promise<CeloTxReceipt>[] = []
+
+    for (let tx of txList) {
+      const promise = tx.sendAndWaitForReceipt({ from: userAccount })
+      promises.push(promise)
+    }
+
+    await Promise.all(promises)
+  }
+
+  test('SBAT vote', async () => {
+    const groupAccount = accounts[0]
+    const validatorAccount = accounts[1]
+    const userAccount = accounts[2]
+
+    await setupGroupAndAffiliateValidator(groupAccount, validatorAccount)
+    await registerAccountWithLockedGold(userAccount)
+    await (await election.vote(groupAccount, ONE_HUNDRED_GOLD)).sendAndWaitForReceipt({
+      from: userAccount,
+    })
+
+    const totalGroupVotes = await election.getTotalVotesForGroup(groupAccount)
+
+    expect(totalGroupVotes).toEqual(ONE_HUNDRED_GOLD)
+  })
+
+  test('SBAT activate vote', async () => {
+    const groupAccount = accounts[0]
+    const validatorAccount = accounts[1]
+    const userAccount = accounts[2]
+
+    await setupGroupAndAffiliateValidator(groupAccount, validatorAccount)
+    await registerAccountWithLockedGold(userAccount)
+    await (await election.vote(groupAccount, ONE_HUNDRED_GOLD)).sendAndWaitForReceipt({
+      from: userAccount,
+    })
+    await mineToNextEpoch(web3)
+
+    const txList = await election.activate(userAccount)
+    let promises: Promise<CeloTxReceipt>[] = []
+    for (let tx of txList) {
+      const promise = tx.sendAndWaitForReceipt({ from: userAccount })
+      promises.push(promise)
+    }
+    await Promise.all(promises)
+
+    const activeVotes = await election.getActiveVotesForGroup(groupAccount)
+    expect(activeVotes).toEqual(ONE_HUNDRED_GOLD)
+  })
+
+  test('SBAT revoke active', async () => {
+    const groupAccount = accounts[0]
+    const validatorAccount = accounts[1]
+    const userAccount = accounts[2]
+
+    await setupGroupAndAffiliateValidator(groupAccount, validatorAccount)
+    await registerAccountWithLockedGold(userAccount)
+    await activateAndVote(groupAccount, userAccount, ONE_HUNDRED_GOLD)
+
+    await (
+      await election.revokeActive(userAccount, groupAccount, ONE_HUNDRED_GOLD)
+    ).sendAndWaitForReceipt({
+      from: userAccount,
+    })
+
+    const remainingVotes = await election.getTotalVotesForGroup(groupAccount)
+    expect(remainingVotes).toEqual(ZERO_GOLD)
+  })
+
+  test('SBAT revoke pending', async () => {
```

Comment:
Below there is a group of unit tests that should be testing `revoke`, this and previous tests are uncategorized. Should they be, per function being tested?

Reply:
wrapped each test in a `describe` to help structure the tests